include/common/isolatable_hardwares.hpp

```diff
@@ -23,35 +23,37 @@ using IsItIsoHwInvPath = bool;
 /**
  * @brief Lookup function signature
```

Comment:
Here I am using a common signature to support different lookups since we need to use different logic for different targets so using the function object a member in the isolatable hardware list and that member will get initialized with the appropriate lookup function based on the targets.

I think, if we added in the below each lookup function, it will look like a common description (you could argue logic will be different but, that is covered as part of function name i.e key alone will change to look for example InstanceId, PrettyName, etc)

Reply:
Confusing also description , and additional definition  before function definition.seems to be confusing.